testing/tests/DevExpress.ui.widgets/actionButton.tests.js

```diff
@@ -0,0 +1,123 @@
+import $ from "jquery";
+import config from "core/config";
+
+import "ui/action_button";
+import "common.css!";
+import "generic_light.css!";
+
+QUnit.testStart(function() {
+    var markup =
+        '<div id="fab-one"></div>\
+        <div id="fab-two"></div>\
+        <div id="fabs"></div>';
+
+    $("#qunit-fixture").html(markup);
+});
+
+const FAB_CLASS = "dx-fa-button";
+const FAB_MAIN_CLASS = "dx-fa-button-main";
+
+QUnit.module("create one action");
```

Comment:
You can use modern syntax and use a function as `QUnit.module` argument to group tests together.

```js
const { test } = QUnit;

QUnit.module("module 1", () => {
  test("new test 1", (assert) => {
    <...>
  });

  test("new test 2", (assert) => {
    <...>
  });
});
```